data-pipeline/fetch_data.py

```
@@ -1004,6 +1004,20 @@ async def fetch_all_data(self):
         
         # Save all data
         self.save_to_database(all_data)
+
+        # Trigger alert generation in the backend
+        if self.use_postgres:
+            try:
+                logger.info(f"[run_id={self.run_id}] Triggering backend alert generation...")
```

Ruff (0.15.6)

[warning] Logging statement uses f-string

(G004)